applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -507,9 +525,6 @@ static int change_active_brdcast_src(void)
 		return ret;
 	}
 
-	/* Always start playing new source */
-	playing_state = true;
```

Comment:
Should we recover the "paused" since that replaced "playing_state "?

Reply:
Not sure what you are thinking here. Can you please elaborate?

Reply:
Since the variable "playing_state" is replaced by "paused" in the code, I was wondering if we should replace line511 to "paused = true;"

Reply:
I think I see.. Since paused is a static, that is by default set to false, so this is not required.

Reply:
The point here was that if you have paused stream_0 and then you switch to stream_1 paused will still be true unless you set 'paused = false' on line 511